File: dayThree/python/question_two.py
def reverseInt(number):
    reverse = 0

    while (number > 0):
        last_digit = number % 10
    
        reverse = (reverse * 10) + last_digit
    
        number = number//10
    return reverse
   
number = 234
print(reverseInt(number))    



#
#number = 234
#def reverseString(number):
#    reverse = ""
#
#    for _ in word:
#        reverse = _ + reverse 
#        
#    return reverse
#        
#word = "234"
#print(reverseString(number))
#

   

File: dayThree/java/QuestionTwo.java
public class QuestionTwo{
    public static void main(String[] args){
   
    int number = 234;
    System.out.println(reversInteger(number)); 
   
    }


    public static int reversInteger(int number){
    int reverse = 0;
    
    while (number != 0){
    int lastDigit = number % 10;
    
    reverse = (reverse * 10) + lastDigit;
    
    number = number/10;
   
    
         }
     return reverse;
         
    }

}


